policy/modules/contrib/l2tp.te

```diff
@@ -30,12 +30,14 @@ allow l2tpd_t self:capability net_admin;
 allow l2tpd_t self:process signal_perms;
 allow l2tpd_t self:fifo_file rw_fifo_file_perms;
 allow l2tpd_t self:netlink_socket create_socket_perms;
+allow l2tpd_t self:netlink_generic_socket create_socket_perms;
 allow l2tpd_t self:rawip_socket create_socket_perms;
 allow l2tpd_t self:socket create_socket_perms;
 allow l2tpd_t self:tcp_socket { accept listen };
 allow l2tpd_t self:unix_dgram_socket sendto;
 allow l2tpd_t self:unix_stream_socket { accept listen };
 allow l2tpd_t self:pppox_socket create_socket_perms;
+allow l2tpd_t sysfs_t:file { open read };
```

Comment:
This line is a subset of dev_read_sysfs() which actually is the correct way to access types from other modules.

Reply:
Sorry, my mistake.

I thought the same from reading the macro definition for _dev_read_sysfs()_, but during development I found I needed the extra "allow" line otherwise I was still hitting some access denials.

However I think I must have gotten my packages mixed up in the test environment as I've just re-tested from scratch without the "allow" and and everything is working.  I'll remove this extra line.